README.md

```diff
@@ -327,6 +327,13 @@ zoom.addEventListeners('show', event => {
 })
 ```
 
+## Compatibility
```

Comment:
It'd be awesome if you add a new line after this heading with new changes coming soon.